### ISSUE TITLE
Airdrop!

### ISSUE THREAD

**Issue by alejandrosanchezm** [keybase/keybase-issues#3722]
I've complete all the requirements (Three green ticks) but after click on "Become a lucky airdropee", the app says "Sorry, you are not qualified to join". Why?
Thank you

**Comment by Stellaruser** [comment]
Exakt das gleiche Problem habe ich auch

**Comment by Derillion455** [comment]
Ja, jeder hat schon verstanden warum. Sie wollen nicht viele Teilnehmer in das Projekt aufnehmen. Nicht alle Länder können teilnehmen. Kleine Länder und solche, die nicht am Badehaus teilnehmen. Und für diejenigen, die teilnehmen - schreibt oft, dass es unmöglich ist, einen Bestätigungscode per SMS zu senden oder auch, dass das Land nicht am airdrop teilnimmt. Niemand, der frei ist, streut dann Geld!

**Comment by Derillion455** [comment]
Ich habe Handynummern für Moldawien und Russland. Sie können sich auch nicht von dort aus anmelden.

**Comment by Stellaruser** [comment]
Ich denke schon das es Betrug ist!! Airdrop ist ein Fake!
 Wenn man sich mal anschaut was dieses Keybase übehaupt ist, da sind ja Google, Facebook & Co ja Waisenknaben dagegen!
Sie wollen nur Daten und alle Verknüpfungen was ihr alles macht. Das ist die ultimative Datensammlung! Der vollkommene gläserne Bürger! Die wissen über Euch besser Bescheid als Ihr selbst!

**Comment by terminalfool** [comment]
I have been on keybase for over three years. I registered for the stellar airdrop after the introductory gift, but have not received any further airdrops. My user page does not have the stellar logo after my name as it should.
